core/src/AzureMcp.Core/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -36,22 +35,28 @@ public static void ConfigureOpenTelemetry(this IServiceCollection services)
 
                 options.IsTelemetryEnabled = string.IsNullOrEmpty(collectTelemetry)
                     || (bool.TryParse(collectTelemetry, out var shouldCollect) && shouldCollect);
-
-                if (options.IsTelemetryEnabled)
-                {
-                    var address = NetworkInterface.GetAllNetworkInterfaces()
-                        .Where(x => x.OperationalStatus == OperationalStatus.Up && x.NetworkInterfaceType != NetworkInterfaceType.Loopback)
-                        .Select(x => x.GetPhysicalAddress().ToString())
-                        .FirstOrDefault(x => !string.IsNullOrEmpty(x));
-
-                    options.MacAddressHash = address != null
-                        ? Sha256Helper.GetHashedValue(address)
-                        : "N/A";
-                }
             });
 
         services.AddSingleton<ITelemetryService, TelemetryService>();
 
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I was just curious why we're not using `Environment.SpecialFolder.UserProfile` for all platforms? which as per documentation resolves to `C:\Users\<uname>` on Win, `/home/<uname>` on Mac and 
`